cloud/k8s/deploy/k8s-deploy.sh

```diff
@@ -67,8 +67,12 @@ function configure_cri_runtime() {
 		configure_containerd
 		;;
 	esac
-	[ -c /host-devices/kvm ] && chmod 666 /host-devices/kvm
-	[ -c /host-devices/kkm ] && chmod 666 /host-devices/kkm
+
+        [ -c /host-dev/kvm ] && chmod 666 /host-dev/kvm
```

Comment:
This should be superfluous, the `udevadm trigger` should trigger the appropriate event and udev itself should change the permissions.

Reply:
What about if udev isn't running at all? Isn't that possible?